docs/quickstart.md

```diff
@@ -0,0 +1,36 @@
+# Quickstart Guide
+
+This guide covers how you can quickly get started using Draft
+
+## Prerequisites
+
+The following prerequisites are required for successful use of Draft.
+
+1. A Kubernetes cluster and the `kubectl` CLI tool
+2. Installation of Helm on your Kubernetes cluster
+3. Installing and configuring Draft on your laptop
+
+### Install Kubernetes or have access to a cluster
+- If you don't already have a running Kubernetes cluster, consider downloading and installing [Minikube][] which can help you get a Kubernetes cluster running on your local machine
```

Comment:
Should we consider also adding Docker for Mac/Windows here, as the Kubernetes feature will probaly get out of the beta channel around DockerCon?

I will test all the steps here with Docker for Mac and maybe add a PR after we know everything works.